week1.css

```
@@ -0,0 +1,138 @@
+header.preview{
+    background-color: rgb(39, 39, 86); 
+    color:beige;
+}
+section.preview{
+    display:flex; 
+    justify-content: space-between; 
+    overflow-x:scroll; 
+}
+section.preview img{
+    width:250px; 
+    height: 350px; 
+    padding-left:10px; 
+    padding-right: 10px;
+}
+nav{
+    text-align:center; 
+    margin:20px 0px 20px 0;  
+    box-sizing: border-box; 
+    padding: 50px 0 50px 0; 
+    border: 2px solid gray;
+    background-color: rgb(188, 168, 184); 
+}
+header.sticky h2{
+    background-color: aquamarine; 
+    color:black;
+}
+div.nav{
+    font-size: 30px; 
+    text-align: center;
+}
+a{
+    text-decoration-line: none; 
+    padding:0 10px 0 10px;
+}
+a:hover {
+    color: yellow; 
+    text-decoration: underline; 
+}
+header.sticky{
+    position: sticky;
+    top:0px;
+}
+div.one{
+    display:flex; 
+    justify-content:center; 
+    flex-wrap: wrap; 
+    margin: 0px 100px 0px 100px;
+}
+img.oneday { 
+    width: 200px; 
+    height: 200px;  
+    padding:10px;
+}
```

얘 같은 경우에는 html에서 img에 모두 일일히 onday를 줘야하는 불편함이 있으니까,
```suggestion
.one > img {
    width: 200px; 
    height: 200px;  
    padding:10px;
}
```
이렇게 하면 좀 더 편리하겠죠?!

---

아하 img.oneday라구 하면 img 마다 oneday라는 class를 줘야하는데 
class='one' 이라는 div태그 안에 모든 img에 주려면 자식 선택자를 해서 주는게 더 나은 방법이라는 거구나!

---

바로 반영!